cmd/db/main.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/moov-io/customers/internal/database"
+)
+
+var flagLogFormat = flag.String("log.format", "", "Format for log lines (Options: json, plain")
+
+type Config struct {
+	RootPassword string `split_words:"true" default:"secret"`
+	User         string `default:"moov"`
+	Password     string `default:"secret"`
+	Address      string `default:"tcp(localhost:3306)"`
+	Database     string `default:"paygate_test"`
+}
+
+func main() {
+	var config Config
+	err := envconfig.Process("mysql", &config)
+	if err != nil {
+		panic(err)
+	}
+
+	err = runCmd(os.Args[1], &config)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func runCmd(cmd string, config *Config) error {
+	switch cmd {
+	case "setup":
+		err := dropDB(config)
```

Comment:
We cannot drop the database. This needs to never be an option in production in fact we will not grant a service `DROP`.

Reply:
For tests this may be useful. For example if you want to test some specific migrations, etc. We can add -force flag for drop and make it work only if database has `_test`.